.eslintrc

```diff
@@ -1,5 +1,6 @@
 {
   "extends": "@appium/eslint-config-appium",
+  "rules": {"import/no-unresolved": "off"},
```

Comment:
is this disabled temporarily?

Reply:
no, see https://github.com/appium/appium-ios/pull/4#issuecomment-868729225

This rule does not work well for a monorepo combined with how we build and publish packages